app-utils/edid-decode/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=edid-decode
+PKGSEC=utils
+PKGDEP=""
```

Comment:
Not possible?

Reply:
It's a simple program with minimal dependencies.

Reply:
Then it would still have at least `glibc`.